templates/helm/values.yaml.tpl

```diff
@@ -16,6 +16,10 @@ deployment:
   labels: {}
   containerPort: 8080
 
+service:
+  create: false
+  type: "ClusterIP"
```

Comment:
@vijtrip2 @RedbackThomson @A-Hilaly

Can we rename this to specify that it has to do with the metrics server and not the service controller itself?

Perhaps something like this?

```yaml
metrics:
  service:
    # Set to true to automatically create a Kubernetes Service resource for the
    # Prometheus metrics server endpoint in controller
    create: false
    # Which Type to use for the Kubernetes Service?
    # See: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
    type: ClusterIP
```

Reply:
Yep, I will get it done.